_episodes/code/02_parallel_jobs/mpi_numpi.py

```diff
@@ -3,15 +3,12 @@
 from mpi4py import MPI
 import numpy as np
 
-np.random.seed(2017)
-
 comm = MPI.COMM_WORLD
 size = comm.Get_size()
 rank = comm.Get_rank()
 
-
 def inside_circle(total_count):
-
+    np.random.seed(2022*rank)
```

Comment:
Aren't you now resetting the seed here every time the function is called? A rank specific seed is indeed necessary...perhaps this deserves an explanatory comment?

Reply:
The function is only called once. Would it be clearer if the seed is set once the rank has been determined, for example on line 9?

Reply:
I think that would be the right thing to do, there are two subtle things going on that should be clarified in a comment (or in the lesson?): you only set the seed once in your application, and the seed of each rank should be different.